src/test/scala/com/stratio/hermes/utils/HermesTest.scala

```diff
@@ -25,54 +29,128 @@ class HermesTest extends FlatSpec with Matchers {
     val hermesEN = Hermes()
     val fullNameEN = hermesEN.Name.fullName
     fullNameEN should fullyMatch regex """[a-zA-Z]+ [a-zA-Z]+"""
-    hermesEN.Name.nameModel.firstNames should contain (fullNameEN.split(" ")(0))
-    hermesEN.Name.nameModel.lastNames should contain (fullNameEN.split(" ")(1))
+    hermesEN.Name.nameModel.firstNames should contain(fullNameEN.split(" ")(0))
+    hermesEN.Name.nameModel.lastNames should contain(fullNameEN.split(" ")(1))
 
     val hermesES = Hermes("ES")
     val fullNameES = hermesES.Name.fullName
     fullNameES should fullyMatch regex """[a-zA-Z]+ [a-zA-Z]+"""
-    hermesES.Name.nameModel.firstNames should contain (fullNameES.split(" ")(0))
-    hermesES.Name.nameModel.lastNames should contain (fullNameES.split(" ")(1))
+    hermesES.Name.nameModel.firstNames should contain(fullNameES.split(" ")(0))
+    hermesES.Name.nameModel.lastNames should contain(fullNameES.split(" ")(1))
   }
 
   it should "generate valid middle names: firstName firstName with EN and ES locales" in {
     val hermesEN = Hermes()
     val middleNameEN = hermesEN.Name.middleName
     middleNameEN should fullyMatch regex """[a-zA-Z]+ [a-zA-Z]+"""
-    hermesEN.Name.nameModel.firstNames should contain (middleNameEN.split(" ")(0))
-    hermesEN.Name.nameModel.firstNames should contain (middleNameEN.split(" ")(1))
+    hermesEN.Name.nameModel.firstNames should contain(middleNameEN.split(" ")(0))
+    hermesEN.Name.nameModel.firstNames should contain(middleNameEN.split(" ")(1))
 
     val hermesES = Hermes("ES")
     val middleNameES = hermesES.Name.middleName
     middleNameES should fullyMatch regex """[a-zA-Z]+ [a-zA-Z]+"""
-    hermesES.Name.nameModel.firstNames should contain (middleNameES.split(" ")(0))
-    hermesES.Name.nameModel.firstNames should contain (middleNameES.split(" ")(1))
+    hermesES.Name.nameModel.firstNames should contain(middleNameES.split(" ")(0))
+    hermesES.Name.nameModel.firstNames should contain(middleNameES.split(" ")(1))
   }
 
   it should "raise an exception when it gets a firstName/lastName and firstNames/lastNames are empty in the locale" in {
     val hermes = Hermes("XX")
     //scalastyle:off
-    an [NoSuchElementException] should be thrownBy hermes.Name.firstName()
-    an [NoSuchElementException] should be thrownBy hermes.Name.lastName()
+    an[NoSuchElementException] should be thrownBy hermes.Name.firstName()
+    an[NoSuchElementException] should be thrownBy hermes.Name.lastName()
     //scalastyle:on
   }
 
   it should "raise an exception when it tries to load a locale that don't exist" in {
     //scalastyle:off
-    val thrown = the [IllegalStateException] thrownBy Hermes("XY").Name.firstName()
+    val thrown = the[IllegalStateException] thrownBy Hermes("XY").Name.firstName()
+    //scalastyle:on
+    thrown.getMessage should equal(s"Error loading locale: /locales/name/XY.json")
+  }
+
+  it should "generate a random integer of 0 digit give it 0" in {
+
```

Comment:
Could you please get rid of blank lines at the beggining and the end of all the tests?